src/modules/redis_cluster/outputs.tf

```
@@ -27,3 +27,8 @@ output "cluster_ssm_path_auth_token" {
   value       = local.ssm_path_auth_token
   description = "SSM path of Redis auth_token"
 }
+
+output "transit_encryption_mode" {
+  value       = local.enabled ? module.redis.transit_encryption_mode : null
```

Module it self should handle the logic
```suggestion
  value       = module.redis.transit_encryption_mode
```